posthog/hogql/database/database.py

```diff
@@ -121,7 +125,9 @@ def add_warehouse_tables(self, **field_definitions: Any):
             setattr(self, f_name, f_def)
 
 
-def create_hogql_database(team_id: int, modifiers: Optional[HogQLQueryModifiers] = None) -> Database:
+def create_hogql_database(
+    team_id: int, modifiers: Optional[HogQLQueryModifiers] = None, team_arg: Optional["Team"] = None
+) -> Database:
```

Comment:
Perhaps we should just pass the `context` here?

Reply:
Yeah, keen to do this - will do so in a follow up PR so that the git blame is better